tools/twix/src/panels/enum_plot.rs

```diff
@@ -97,17 +97,27 @@ pub struct EnumPlotPanel {
 impl Panel for EnumPlotPanel {
     const NAME: &'static str = "Enum Plot";
 
-    fn new(nao: Arc<Nao>, _value: Option<&serde_json::Value>) -> Self {
-        Self {
+    fn new(nao: Arc<Nao>, value: Option<&Value>) -> Self {
+        let output_key = value.and_then(|value| String::from(value.get("subscribe_key").as_str())).unwrap_or_default();
```

Comment:
whoops, this should do it
```suggestion
        let output_key = value
            .and_then(|value| value.get("subscribe_key"))
            .and_then(|value| value.as_str())
            .map(String::from)
            .unwrap_or_default();

```